platforms/js/build_js.py

```diff
@@ -194,10 +199,25 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
+    parser.add_argument('--cmake_option', action='append', help="Adds the ability to modify the cmake list")
+    parser.add_argument('--build_flags',action='append',help="prepare flags for building")
+    parser.add_argument('--whitelist', help="Add an ability to connect a JSON file that include list of the OpenCV functions needed for developing a project")
```

Comment:
```.diff
-whitelist
+config
```

'Specify configuration file with own list of exported into JS functions'

Also it would be better to move current stuff from .py code into `opencv_js.json`.

BTW, You can take a look on Android config files (which allows Python code inside).